### PR TITLE
minor formatting and text updates

### PR DESCRIPTION
**Pull request by fmcdonal**

### CODE REVIEW

**Comment by wtrocki** [comment]
Looks good!

File: docs/kafka/nodejs/README.adoc [hunk 1]
```diff
@@ -229,7 +229,7 @@ $ npm install
 $ node consumer.js
 ----
 +
-You should see the Node.js application start to run and connect to the Kafka instance. However, because you haven't yet run the producer component, the consumer has no country names to display.
+You should see see the Node.js run and connect to the Kafka instance. However, because you haven't yet run the producer component, the consumer has no country names to display.
```
Comment (jbyrne-redhat) [inline docs/kafka/nodejs/README.adoc:232]: @fmcdonal Repeat of the word 'see' here.

Also, it should read: "You should see the Node.js _application_ run...". This one is my fault :) I wrote it incorrectly in the JIRA, which I just updated.

**Review by jbyrne-redhat — CHANGES_REQUESTED** [review]
Looks great, @fmcdonal  - just two very minor updates to request.

**Comment by fmcdonal** [comment]
@jbyrne-redhat changes applied

**Review by jbyrne-redhat — APPROVED** [review]
Looks great, @fmcdonal  - thanks! Ready to merge.

**Pull request merged**